### DISCUSSION TITLE
Question: OpenOrienteering Mapper in Chocolatey repository

### DISCUSSION THREAD

**Discussion by jmacura** [OpenOrienteering/mapper#1447] [Questions & Answers] [answered]
### Context
I've started to use [Chocolatey](https://chocolatey.org/) recently. For my own surprise, I have found OpenOrienteering Mapper in the repository. Last version available there is 0.9.0 as of the time of writing this. 

### Question
Is the Mapper installer in choco an official release maintained by OpenOrienteering? Is it updated with new releases?

### Configuration
Mapper Version: latest release
Operating System: Windows supported versions

**Comment by sikmir** [comment, marked as answer]
[OpenOrienteering Mapper](https://chocolatey.org/packages/openorienteering-mapper) choco package maintained by @chtof ([package sources](https://github.com/chtof/chocolatey-packages/tree/master/automatic/openorienteering-mapper)). Strictly speaking, that's not an official one, though official release asset is used there.

**Comment by chtof** [comment]
Updates are automated but sometimes a delay is needed.
I see the version 0.9.1 has been released only yesterday. New version has been correctly detected but there was a failure during an upload of the new version: https://gist.github.com/chtof/393704c64c2344f0f8e029cd57bebd2b
An attempt is executed every 8 hours.

**Comment by jmacura** [comment]
@sikmir Thanks for clarification
@chtof Thanks for the info. Great to hear that it's maintained and automated.

**Comment by dg0yt** [comment]
> Is the Mapper installer in choco an official release maintained by OpenOrienteering?

No, it isn't. However, AFAIU the current Chocolatey package will just pull the official binaries that we build and publish.

**Comment by jmacura** [comment]
My question is answered, updates seems to be delivered (0.9.1 arrived just some week later), thus I am closing the issue.
